scripts/run_posix_docker.sh

```diff
@@ -120,10 +120,10 @@ fi
 
 docker run --rm \
     --name test_worker_agent \
-    -v $(pwd):/code:ro \
-    -v ${HOME}/.aws:/aws \
-    --env-file ${TMP_ENV_FILE} \
-    ${ARGS} \
+    -v "$(pwd)":/code:ro \
+    -v "${HOME}"/.aws:/aws \
+    --env-file "${TMP_ENV_FILE}" \
+    "${ARGS}" \
```

Comment:
```suggestion
    ${ARGS} \
```

I don't think that this one should be quotes. `ARGS` is built-up in this script and contains multiple command-line arguments appended together. Quoting this turns that list of args into a single argument.

Reply:
Fixed